src/features/Settings/components/AddOrEditNetworkNode.test.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+
+import { MemoryRouter } from 'react-router-dom';
+import {
+  actionWithPayload,
+  fireEvent,
+  mockUseDispatch,
+  ProvidersWrapper,
+  simpleRender,
+  waitFor
+} from 'test-utils';
+
+import { fAccounts, fAssets, fContacts, fNetwork, fNetworks, fSettings } from '@fixtures';
+import { DataContext, NetworkUtils, ProviderHandler, StoreProvider } from '@services';
+import { translateRaw } from '@translations';
+import { NodeType } from '@types';
+import { noOp } from '@utils';
+
+import AddOrEditNetworkNode from './AddOrEditNetworkNode';
+
+// Mock getCurrentBlock
+ProviderHandler.prototype.getCurrentBlock = jest.fn().mockResolvedValue(11386255);
```

Comment:
I believe this mocks the method for the entire test suite.

Reply:
Yes, which is intended.